ui/analyse/src/study/studyComments.ts

```diff
@@ -63,21 +63,6 @@ export function currentComments(ctrl: AnalyseCtrl, includingMine: boolean): VNod
               ),
             })
           : null,
-        isMine && study.vm.mode.write
-          ? h('a.edit', {
-              attrs: {
-                'data-icon': '',
-                title: 'Edit',
-              },
-              hook: bind(
-                'click',
-                _ => {
-                  study.commentForm.start(chapter.id, ctrl.path, node);
-                },
-                ctrl.redraw
-              ),
-            })
-          : null,
```

Comment:
This was completely unused as far as I can tell. There is no way to be in "show my comments in the list" mode while also being able to edit comments.

In theory, it could maybe appear via this call:
https://github.com/ornicar/lila/blob/0c21c89fa60a636d7c00058862ce39d50a177301/ui/analyse/src/study/commentForm.ts#L98

But I'm pretty sure that call is also unreachable as it is since comments are never shown without calling `commentForm.start` which means `current` will never be `null` here. But I guess keeping this as a safety check is ok since `viewDisabled` is needed elsewhere anyway.